Cartfile

```diff
@@ -1,6 +1,7 @@
-binary "https://www.mapbox.com/ios-sdk/Mapbox-iOS-SDK.json" ~> 4.3
+binary "https://www.mapbox.com/ios-sdk/Mapbox-iOS-SDK.json" ~> 5.8
+github "mapbox/MapboxGeocoder.swift" ~> 0.10
```

Comment:
Hier nog iets interessants veranderd in die versie bump?

Reply:
Deze versie gebruikte wij al alleen de oude nav sdk natuurlijk niet dus voor ons geen wijzigingen